### DISCUSSION TITLE
Recap on FAIRmat project meeting and users meeting, Next topics in TF

### DISCUSSION THREAD

**Discussion by budschi** [FAIRmat-NFDI/AreaA-data_modeling_and_schemas#76] [Task Force Meeting]
# Project meeting

* overall good meeting
* last day too long
* weak: follow up and next steps, what is expected, how to proceed from current status
* better prepare what are next steps and what needs to be done until next meeting
* good: interarea meeting topics and discussion
* critical: DFG metrics on measuring project results, needs definition
* define small groups that work out specific topics (e.g. data privacy) --> needs specialists!
* good: lots of raised questions
* good to have it in IRIS
* good: other restaurants
* bad: Hotel booking

# Users meeting
* good:
  * talks
  * number of user
  * active discussion
  * interested users attended, ready to start using our tools
  * posters: lots of discussions, users were very interested, next to venue, --> should be continued like this, + standing table for life demo
  * direct transition to dinner
  * tutorials/workshops of Area A&D
  * long enough programm to make it worth travelling to Berlin!
* bad:
  * nothing
  * 
* ideas:
  * more demos?
  * interactive workshop was better than online tutorials

# next steps
* publication strategy --> each area should publish an article! important for DFG! internal review system
* Castel'23: Tamas is going, if it is really important Hampus might go
* Leibinz Meeting on Research data management
* 14.12. 10:00h
*

# next TF topics
* elabFTW - NOMAD: Tamas, Fabian, Amir, --> 27.11.
* data modelling for layer growth: --> NOMAD materials processing (PLD, PVD, MOVPE, MBE) (4.12.)
* writing a NOMAD archive from other ELNs --> how to export into NOMAD and how to automatize
* data modelling for crystal growth: 
* PPMS from Dresden including labfolder integration
* Harmonizing Task 1 and 2 data models
*

**Comment by lapmk** [comment]
I Couldn't have said it better! I fully agree with the recap.
